app/controllers/place_controller.rb

```diff
@@ -14,12 +12,15 @@ class PlaceController < ApplicationController
   REPORT_CHILD_ABUSE_SLUG = "report-child-abuse-to-local-council".freeze
 
   def show
+    set_content_item(PlacePresenter)
+
     if request.post?
       @location_error = location_error
       if @location_error
         @postcode = postcode
       elsif imminence_response.places_found?
-        @publication = PublicationWithPlacesPresenter.new(artefact, imminence_response.places)
+        @publication = PlacePresenter.new(content_item, imminence_response.places)
+        set_language_from_publication
```

Comment:
Oh right! But do we need this second call to set the language at all then? Presumably the content item it's referring to is still the same so the language wouldn't have changed?